reference arbitrum/sdk in docs (#2121)
OffchainLabs/arbitrum: docs/Frontend_Integration.md

## Arbitrum / Ethereum Bridge

> Accessing bridging methods can be done via our `@arbitrum/sdk` library, or by simply connecting to the relevant contracts directly.

Reviewer: ```suggestion
Accessing bridging methods can be done via the `Arbitrum SDK`, or by simply connecting to the relevant contracts directly.
```

Reviewer: ```suggestion
Accessing bridging methods can be done via the [`Arbitrum SDK`](https://github.com/OffchainLabs/arbitrum-sdk) library, or by simply connecting to the relevant contracts directly.
```

> #### @arbitrum/sdk

Reviewer: ```suggestion
#### Arbitrum SDK
```
